lib/liquid/combinators/general.ex

```diff
@@ -166,6 +166,10 @@ defmodule Liquid.Combinators.General do
     |> traverse({__MODULE__, :to_atom, []})
   end
 
+  @type condition :: [
+          condition: {LexicalToken.value(), Atom.t(), LexicalToken.value()}
```

Comment:
Not every `atom` can be the second element in the tuple. This limited number of atoms must be represented by a type called `comparison_operator` or something like this